CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ option(MMDEPLOY_BUILD_SDK_CSHARP_API "build SDK C# API support" OFF)
 option(MMDEPLOY_BUILD_SDK_JAVA_API "build SDK JAVA API" OFF)
 option(MMDEPLOY_BUILD_EXAMPLES "build examples" OFF)
 option(MMDEPLOY_SPDLOG_EXTERNAL "use external spdlog" OFF)
-option(MMDEPLOY_ZIP_MODEL "support SDK model in zip format" OFF)
+option(MMDEPLOY_ZIP_MODEL "support SDK model in zip format" ON)
```

Comment:
It is recommended to keep `MMDEPLOY_ZIP_MODEL` `OFF` as the default value because not all of the users need it